docs/source/AdministratorGuide/ServerInstallations/environment_variable_configuration.rst

```diff
@@ -5,9 +5,19 @@ Environment Variables to Configure DIRAC Behaviour
 ==================================================
 
 There is a small number of environment variables that can be set to control the behaviour of some DIRAC
-components. These variables can either be set in the ``bashrc`` file of a client or server installation or set manually
+components. These variables can either be set in the ``bashrc`` file of a **client or server** installation or set manually
 when desired.
 
+DIRAC_VOMSES
```

Comment:
Is there a reason to move this variable to the front?
The list of variables was initially sorted alphabetically, but I already dropped the ball when `DIRAC_M2CRYPTO_SPLIT_HANDSHAKE` was added.

Reply:
My only reason to move it in front is because it's a more "common" environment variable

Reply:
I think they should be in alphabetical order ! Sorry if I screwed up with the SPLIT_HANDSHAKE, I did not even notice your comment. Since you are at it, can you please put it in the correct place too ?

Reply:
I hadn't commented in the other PR... You miss one review...